src/plugins/cloudinary/models/video-source/video-source.js

```diff
@@ -187,7 +187,7 @@ class VideoSource extends BaseSource {
 
   generateSources() {
     if (this.isRawUrl) {
-      const type = this.sourceTypes().length > 1 ? null : this.sourceTypes()[0];
+      const type = this.sourceTypes()[0] === 'auto' ? null : this.sourceTypes()[0];
```

Comment:
This should return `null` for raw-urls, but our new default `sourceTypes` is `['auto']` so this returns `this.sourceTypes()[0]` which causes the issue.

This change fixes that, but breaks a test that expects
```http://res.cloudinary.com/demo/video/upload/vc_vp9/sea_turtle.webm```
to be returned when just `video_codec: 'vp9'` transformation is set
and we are now (since moving to `f_auto` by default) returning
```https://res.cloudinary.com/demo/video/upload/vc_vp9/f_auto:video/elephants```